rotkehlchen/chain/ethereum/tokens.py

```diff
@@ -199,25 +229,19 @@ def _detect_tokens(
             exceptions=exceptions,
             except_protocols=['balancer'],
         )
-        for account in accounts:
-            if self.ethereum.connected_to_any_web3():
-                querying_etherscan = False
-                tokens_batch_size = OTHER_MAX_TOKEN_CHUNK_LENGTH
-            else:
-                querying_etherscan = True
-                tokens_batch_size = ETHERSCAN_MAX_TOKEN_CHUNK_LENGTH
-            token_balances = self._query_chunks(
-                account=account,
-                chunks=get_chunks(all_tokens, n=tokens_batch_size),
-                querying_etherscan=querying_etherscan,
-            )
+
+        queried_balances = self._chunk_and_query(
+            accounts=accounts,
+            tokens=all_tokens,
+        )
+        for account, token_balances in queried_balances.items():
             detected_tokens = list(token_balances.keys())
             self.db.save_tokens_for_address(write_cursor, account, detected_tokens)
 
     def query_tokens_for_addresses(
             self,
-            addresses: List[ChecksumEthAddress],
-    ) -> TokenBalancesType:
+            accounts: List[ChecksumEthAddress],
```

Comment:
Why the semantic change of accounts -> addresses?

Reply:
`accounts` name is actually mostly used in the other places where this method is called. Plus, `detect_tokens` already uses `accounts`. Just wanted to be consistent with this variable name

Reply:
Addresses is a better term to use. Since it's an ethereum address and not an account.

Reply:
Also in this same file addresses is used. So basically your change is breaking consistency, not helping it.